### ISSUE TITLE
drawer-dialog: color contrast on handle

### ISSUE THREAD

**Issue by agliga** [eBay/skin#1696]
Drawer dialog's handle does not meet correct color contrast. Need to change color from grey 4 to 5

**Comment by agliga** [comment]
Published in `skin@13.5.1`
